src/components/combo_box/combo_box.test.tsx

```diff
@@ -221,6 +221,30 @@ describe('props', () => {
 
     expect(component).toMatchSnapshot();
   });
+
+  test('icon is rendered', () => {
+    const component = shallow(
+      <OuiComboBox
+        options={options}
+        selectedOptions={[options[2]]}
+        icon={true}
+      />
+    );
+
+    expect(component).toMatchSnapshot();
```

Comment:
The resulting snapshots int the two tests here only show that the icon prop is set, It does not test the rendered html. The goal is to make sure that the icon has rendered, so we should explicitly check for that.